src/Collection.php

```diff
@@ -812,6 +901,24 @@ public function listIndexes(array $options = [])
         return $operation->execute(select_server($this->manager, $options));
     }
 
+    /**
+     * Returns information for all Atlas Search indexes for the collection.
+     * Only available when used against a 7.0+ Atlas cluster.
```

Comment:
I realize this PR doesn't introduce any new documentation. We typically add code and docs in the same PR for PHPLIB, but I don't want to hold this up for that. I've added a `documentation-needed` label to [PHPLIB-1143](https://jira.mongodb.org/browse/PHPLIB-1143) so we can follow up with that later.

Reply:
Thanks.

Reply:
Starting the example: https://github.com/mongodb/mongo-php-library/pull/1147